src/components/address-input/_macro.spec.js

```diff
@@ -416,13 +344,15 @@ describe('macro: address-input', () => {
                 }),
             );
 
-            expect($('.ons-js-address-manual-btn').attr('href')).toBe('https://example.com/edit-address');
-            expect($('.ons-js-address-manual-btn').text().trim()).toBe('Manually enter address');
+            test('THEN: renders `manualLinkText` with `manualLink`', () => {
```

Comment:
I think the wording of this test could be more descriptive on what it is testing